src/interfaces/routes.ts

```diff
@@ -120,6 +120,23 @@ export interface PlaceOrderResponse extends OrderResponse {
   postOnly?: boolean;
 }
 
+export interface OrderSettlementUpdateResponse {
+  orderHash: string;
+  userAddress: string;
+  symbol: string;
+  message: string;
+  quantitySentForSettlement: string;
+  orderQuantity: string;
```

Comment:
I remember we  added `orderQuantity` in requeued payload as well ?

Reply:
orderQuantity is the total quantity of the order, I have started sending it from OME but haven't catered on Dapi yet that's why didn't add that here.